Source/Editor/Options/InterfaceOptions.cs

```diff
@@ -341,11 +362,11 @@ public FontReference OutputLogTextFont
         public bool OutputLogScrollToBottom { get; set; } = true;
 
         /// <summary>
-        /// Gets or sets a value indicating whether auto-focus game window on play mode start.
+        /// Gets or sets a value indicating what panel should be focused when play mode start.
         /// </summary>
-        [DefaultValue(true)]
-        [EditorDisplay("Play In-Editor", "Focus Game Window On Play"), EditorOrder(500), Tooltip("Determines whether auto-focus game window on play mode start.")]
-        public bool FocusGameWinOnPlay { get; set; } = true;
+        [DefaultValue(PlayModeFocus.GamePanel)]
+        [EditorDisplay("Play In-Editor", "Focus On Play"), EditorOrder(500), Tooltip("Set what panel to focus on play mode start.")]
+        public PlayModeFocus FocusOnPlayMode { get; set; } = PlayModeFocus.GamePanel;
```

Comment:
I think it would be nice to leave auto-upgrader for old option so people who disabled it will get the same logic. Try adding:
```cs
// [Deprecated in v1.10]
[Serialize, Obsolete, NoUndo]
private bool FocusGameWinOnPlay
{
    get => throw new Exception();
    set
    {
        // Upgrade value
        FocusOnPlayMode = value ? PlayModeFocus.GamePanel : PlayModeFocus.None;
    }
}
```

Reply:
Added in https://github.com/FlaxEngine/FlaxEngine/pull/2994/commits/88215ca3f0e3e54ce8a29db1b5bfdde2c476f494.

Thanks for the reviews :)